foundation-models/openai/src/main/java/com/sap/ai/sdk/foundationmodels/openai/model/OpenAiChatCompletionOutput.java

```diff
@@ -40,9 +40,6 @@ public class OpenAiChatCompletionOutput extends OpenAiCompletionOutput
    */
   @Nonnull
   public String getContent() throws OpenAiClientException {
-    if (getChoices().isEmpty()) {
-      return "";
-    }
```

Comment:
(Question)

What is the reason to remove this check? Would you rather risk a `NullPointerException´?

Reply:
> You cannot have 0 choices per OpenAI docs.

I understand, but for me this would rather justify an exception than ignoring it :)

Not a strong opinion.

Reply:
Charles and I decided on reading this and adding tests.